client/src/components/FoodSeeker/SearchResults/StakeholderDetails/StakeholderDetails.js

```diff
@@ -287,7 +299,7 @@ const StakeholderDetails = ({ onBackClick, isDesktop }) => {
           </Grid2>
           <Grid2 xs={10}>
             <Stack direction="column" xs={10}>
-              <Stack>
+              <Stack width="100%">
```

Comment:
The specified width on this Stack component appears to be redundant.

Reply:
fixed